src/index.html

```diff
@@ -11,9 +11,221 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
 </head>
 <body>
-
+  <header class="header">
+    <div class="menu-logo">
+      <a href="#" class="logo"><img src="images/logo.png" alt="Moyo logo"></a>
+    </div>
+    <nav class="menu-container">
```

Comment:
Меню на странице может быть не одно, плюс ты не используешь этот класс вообще
```suggestion
    <nav class="nav-container">
```